src/main/java/uk/gov/companieshouse/account/validator/service/FelixAccountValidator.java

```diff
@@ -50,31 +52,44 @@ public FelixAccountValidator(Logger logger,
     /**
      * Validate submits the file to the felix validator for validation
      *
-     * @param file An IXBRL file or a ZIP file containing the accounts
+     * @param file        An IXBRL file or a ZIP file containing the accounts
+     * @param fileContent Object contains information required for account/package
+     *                    related requirement
      * @return the result of the validation
      */
     @Override
-    public void startValidation(FileDetailsApi file) throws XBRLValidationException {
+    public void startValidation(FileDetailsApi file, AccountsDetails fileContent) throws XBRLValidationException {
         String fileId = file.getId();
         String callbackUrl = getCallbackUrl(file.getId());
-
+        PackageTypeApi packageType = fileContent.getPackageType();
+        AsyncValidationRequestApi asyncValidationRequestApi = new AsyncValidationRequestApi(fileId, callbackUrl,
+                packageType);
```

Comment:
Can we do this in one line like: 
```suggestion
        AsyncValidationRequestApi asyncValidationRequestApi = new AsyncValidationRequestApi(fileId, callbackUrl,
                fileContent.getPackageType());
```